### PR TITLE
Add multiple rows remove functions

### PR DESCRIPTION
**Pull request by jackyzy823**
For #338

### CODE REVIEW

File: src/ui/code/settingsPage.ts [hunk 1]
```diff
@@ -208,7 +212,7 @@ export class settingsPage {
 		settingsPage.grdServers = jq("#grdServers").DataTable({
 			"dom": dataTableCustomDom,
 			paging: true,
-			select: true,
+			select: { style: "multi+shift" },
```
Comment (salarcode) [inline src/ui/code/settingsPage.ts:215]: With this select style , the edit updates are not reflected on the grid. Also the behavior is weird.
Select "os" works ok I think

File: src/ui/settings.html [hunk 2]
```diff
@@ -712,6 +718,10 @@ <h3 class="mt-1">
 									data-localize="settingsAddRulesSubscriptionButton">
 									Subscribe to a rules list
 								</button>
+								<button id="btnRemoveMultipleRulesSubscription" class="btn btn-info" style="display: none;"
```
Comment (salarcode) [inline src/ui/settings.html:721]:
```suggestion
								<button id="btnRemoveMultipleRulesSubscription" class="btn btn-danger" style="display: none;"

```

File: src/_locales/default-messages.json [hunk 3]
```diff
@@ -488,6 +491,9 @@
 	"settingsConfirmRemoveProxyServer": {
 		"message": "Are you sure to remove the selected proxy server?"
 	},
+	"settingsConfirmRemoveMultipleProxyServer": {
+		"message": "Are you sure to remove the selected multiple proxy servers?"
```
Comment (salarcode) [inline src/_locales/default-messages.json:495]:
```suggestion
		"message": "Are you sure to remove the selected proxy servers?"
```

File: src/_locales/default-messages.json [hunk 4]
```diff
@@ -515,6 +521,9 @@
 	"settingsConfirmRemoveProxyRule": {
 		"message": "Are you sure to remove the selected rule?"
 	},
+	"settingsConfirmRemoveMultipleProxyRule": {
+		"message": "Are you sure to remove the selected multiple rules?"
```
Comment (salarcode) [inline src/_locales/default-messages.json:525]:
```suggestion
		"message": "Are you sure to remove the selected rules?"
```

File: src/_locales/default-messages.json [hunk 5]
```diff
@@ -935,6 +950,9 @@
 	"settingsConfirmRemoveServerSubscription": {
 		"message": "Are you sure to remove the selected subscription?"
 	},
+	"settingsConfirmRemoveMultipleServerSubscription": {
+		"message": "Are you sure to remove the selected multiple subscriptions?"
```
Comment (salarcode) [inline src/_locales/default-messages.json:954]:
```suggestion
		"message": "Are you sure to remove the selected subscriptions?"
```

File: src/_locales/default-messages.json [hunk 6]
```diff
@@ -1064,6 +1085,9 @@
 	"settingsConfirmRemoveRulesSubscription": {
 		"message": "Are you sure to remove the selected rules subscription?"
 	},
+	"settingsConfirmRemoveMultipleRulesSubscription": {
+		"message": "Are you sure to remove the selected mutliple rules subscriptions?"
```
Comment (salarcode) [inline src/_locales/default-messages.json:1089]:
```suggestion
		"message": "Are you sure to remove the selected rules subscriptions?"
```

**Review by salarcode — CHANGES_REQUESTED** [review]

File: src/ui/code/settingsPage.ts (hunk 1 above)
Comment (jackyzy823) [reply src/ui/code/settingsPage.ts]: With "os" style, user might not know using "Ctrl" and "shift" to select multiple rows. :( 

(maybe a line of hint would better)

**Comment by salarcode** [comment]
Looks good to me, thanks

**Comment by salarcode** [comment]
I will move the buttons to the bottom of page next to clear and add some hint for how to use them

**Pull request merged**